alpine-tinyauth installation is crashing

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

alpine-tinyauth

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/alpine-tinyauth.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Alpine

### 📝 Provide a clear and concise description of the issue.

Tried using the helperscript for tinyauth on my machine.
Tested it with deafault and also advanced setting.
Always crashing when promted to enter the subdomain for the service. Entering `https://my.domain.com` or only `my` or nothing always crashes the script with the same error.

### 🔄 Steps to reproduce the issue.

1. open shell on proxmox
2. paste `bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/alpine-tinyauth.sh)"` and hit enter
3. installing via default settings
4. getting promted to enter subdomain for tinyauth
5. entering `https://my.domain.com` or only `my` or nothing
6. getting crash error of the script

### ❌ Paste the full error output (if available).

```
Enter your Tinyauth subdomain (e.g. https://tinyauth.example.com): 
   Creating Tinyauth Service0: line 52: line: unbound variable

[ERROR] in line 52: exit code 1: while executing command cat <<EOF > /etc/init.d/tinyauth
#!/sbin/openrc-run
description="Tinyauth Service"

command="/opt/tinyauth/tinyauth"
directory="/opt/tinyauth"
command_user="root"
command_background="true"
pidfile="/var/run/tinyauth.pid"

start_pre() {
    if [ -f "/opt/tinyauth/.env" ]; then
        while IFS= read -r line || [ -n "$line" ]; do
            [ -z "$line" ] && continue
            case "$line" in
                '#'*)
                    continue
                    ;;
            esac
            export "$line"
        done < "/opt/tinyauth/.env"
    fi
}

depend() {
    use net
}
EOF


/dev/fd/63: line 74: SPINNER_PID: unbound variable
```

### 🖼️ Additional context (optional).

I found this PR which sounds like fixing this error but it haven't it seems: https://github.com/community-scripts/ProxmoxVE/pull/4901

--- a/install/alpine-tinyauth-install.sh
+++ b/install/alpine-tinyauth-install.sh
@@ -49,7 +49,7 @@ EOF
 
 sed -i -e 's/\$/\$\$/g' /opt/tinyauth/.env
 
-cat <<EOF >/etc/init.d/tinyauth
+cat <<'EOF' >/etc/init.d/tinyauth
 #!/sbin/openrc-run
 description="Tinyauth Service"
 

